ProcessMaker/Policies/ProcessRequestPolicy.php

```diff
@@ -33,29 +33,8 @@ public function before(User $user)
      */
     public function view(User $user, ProcessRequest $processRequest)
     {
-        if ($processRequest->user_id == $user->id) {
```

Comment:
Seems this permission check is not covered neither by `ProcessRequestPolicyTest` nor `ForUserScopeTest`, please add a test for this.

Reply:
@caleeli ,

Correct me if I'm wrong, but doesn't the [scopeUserStarted](https://github.com/ProcessMaker/processmaker/blob/bugfix/FOUR-7325/ProcessMaker/Traits/ForUserScope.php#L32) do exactly that?

```php
public function scopeUserStarted($query, $user)
{
    return $query->where('user_id', $user->id);
}

...
return ProcessRequest::forUser($user)
            ->where('process_requests.id', $processRequest->id)
            ->exists();
 ```

Reply:
@paulosakamoto I talked to David about this on slack and he suggested we split the test into its own test so I did that